africa/README.md

```diff
@@ -1,3 +1,4 @@
 # AFRICA
+-[CAPE VERDE](../cape_verde)
```

Comment:
`..` means the level above, but `cape_verde` directory is hanging from `africa`. Also, it should point to its readme file, rather than the directory.